parquet/src/main/java/org/apache/iceberg/parquet/ParquetWriter.java

```diff
@@ -185,9 +190,17 @@ public List<Long> splitOffsets() {
     return null;
   }
 
+  private long estimateBufferSize() {
```

Comment:
Rename `estimateBufferSize` to `estimateBufferedSize` for naming consistency.

Reply:
Also, can we add a few comments about the estimation logic? i.e. what the ratio `totalRowGroupSize / totalBufferSize` means as estimating the ratio of size reduction to match closely what will be written out encoded/compressed.